src/meta/ConfigManager.h

```diff
@@ -0,0 +1,200 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef META_CONFIGMANAGER_H_
+#define META_CONFIGMANAGER_H_
+
+#include "base/Base.h"
+#include <gtest/gtest_prod.h>
+#include "meta/client/MetaClient.h"
+
+namespace nebula {
+namespace meta {
+
+class ConfigManager {
+    FRIEND_TEST(ConfigManTest, MetaConfigManTest);
+    FRIEND_TEST(ConfigManTest, MemConfigManagerTest);
+
+public:
+    static std::unique_ptr<ConfigManager> instance(MetaClient* client = nullptr);
+
+    virtual ~ConfigManager();
+
+    virtual folly::Future<StatusOr<bool>> setConfig(folly::StringPiece space,
```

Comment:
Why use Future?

Reply:
These interface is all for console which goes to meta server, I will add some interface for code and get/set from cache.